src/amo/css/inc/vars.scss

```diff
@@ -30,6 +30,10 @@ $breakpoints: (
     (
       min-width: 500px,
     ),
+  mediumcard: only screen and
```

Comment:
I don't think we should be adding new breakpoints. Can we use the existing ones?

Reply:
Yep, looks good with `medium`.

Reply:
The issue dealt with increasing the readability of this view(German):
https://user-images.githubusercontent.com/31961530/34110069-36ece7dc-e40e-11e7-9404-4518457dff13.png
If I remove the breakpoint, I am afraid that there would be no other view for viewports between 720px and 760px.
I can add media queries for the same but that doesn't go well with the rest of the code flow.

Reply:
ah, then use `extraLarge`.

Reply:
done :+1: